src/PrestaShopBundle/EventListener/UserLocaleListener.php

```diff
@@ -26,20 +26,36 @@
 
 namespace PrestaShopBundle\EventListener;
 
+use Context;
```

Comment:
Could not find any usage of this class inside the class, where is it used ?

Reply:
In the phpdoc of $prestaShopContext